loopchain/channel/channel_inner_service.py

```diff
@@ -637,7 +637,9 @@ async def block_request(self, request_from: str, height: int):
         Note that BlockRequest allows only Block 1.0+
         """
         blockchain = self._channel_service.block_manager.blockchain
-        block: Block_V1_0 = blockchain.find_block_by_height(height)
+        block: Optional[Block_V1_0] = blockchain.find_block_by_height(height)
+        if not block:
+            return  # FIXME: Ignore response? or Send client to failure message?
```

Comment:
Please comment on this.

Reply:
I think it needs CommonReply with Fail code.

Reply:
I mean we have to change BlockRequest of peer_outer to treat in this case.

Reply:
# FIXME -> # This case have to handle by the requesting node.

Reply:
Fixed.